### PR TITLE
upgraded OpenResty to 1.11.2.3.

### PR DESCRIPTION
**Pull request by moonming**
follow the new version of OpenResty : https://openresty.org/en/changelog-1011002.html

### CODE REVIEW

**Comment by denji** [comment]
homebrew/nginx/openresty-debug:
  * 'revision 2' should be removed
homebrew/nginx/openresty
  * 'revision 2' should be removed

File: Formula/openresty-debug.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class OpenrestyDebug < Formula
   desc "Scalable Web Platform by Extending NGINX with Lua"
   homepage "http://openresty.org"
-  VERSION = "1.11.2.2".freeze
+  VERSION = "1.11.2.3".freeze
   url "https://openresty.org/download/openresty-#{VERSION}.tar.gz"
-  sha256 "7f9ca62cfa1e4aedf29df9169aed0395fd1b90de254139996e554367db4d5a01"
+  sha256 "7a0a8570fd3eb193913eb2287f7c926b47e363f376e80c7aa332c35d0fccde69"
   revision 2
```
Comment (denji) [inline Formula/openresty-debug.rb]: homebrew/nginx/openresty-debug:
  * 'revision 2' should be removed

File: Formula/openresty.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Openresty < Formula
   desc "Scalable Web Platform by Extending NGINX with Lua"
   homepage "http://openresty.org"
-  VERSION = "1.11.2.2".freeze
+  VERSION = "1.11.2.3".freeze
   url "https://openresty.org/download/openresty-#{VERSION}.tar.gz"
-  sha256 "7f9ca62cfa1e4aedf29df9169aed0395fd1b90de254139996e554367db4d5a01"
+  sha256 "7a0a8570fd3eb193913eb2287f7c926b47e363f376e80c7aa332c35d0fccde69"
   revision 2
```
Comment (denji) [inline Formula/openresty.rb]: homebrew/nginx/openresty
  * 'revision 2' should be removed

**Comment by moonming** [comment]
@denji removed revision. thanks.

**Pull request merged**